src/main/assets/en_about.html

```diff
@@ -1,36 +1,32 @@
-Web page
-<br/>&nbsp;&nbsp;<a href="https://groups.google.com/d/forum/whereyougo">https://groups.google.com/d/forum/whereyougo</a>
-<br/>&nbsp;&nbsp;<a href="mailto:whereyougo@googlegroups.com/">whereyougo@googlegroups.com</a>
-<br/>&nbsp;&nbsp;<a href="https://github.com/biylda/WhereYouGo/blob/master/PRIVACY.md">Privacy policy</a>
+<b>Related web pages:</b>
+<br/>
+<br/>&nbsp;&nbsp;<a href="https://groups.google.com/d/forum/whereyougo">Google Group mailing list webpage</a>
+<br/>&nbsp;&nbsp;<a href="mailto:whereyougo@googlegroups.com/">Direct mail to whereyougo@googlegroups.com</a>
```

Comment:
Is "c:geo" now owner or manager of this google group?

Reply:
Nope, this is a group which existed before.
I only subscribed to that group and set up a job to pull these mails to our support system, in order to be able to answer them.

We can think about setting up an own contact address at a later point of time.